crates/telio-relay/src/derp/http.rs

```diff
@@ -259,7 +259,7 @@ async fn connect_http<R: AsyncRead + Unpin, W: AsyncWrite + Unpin>(
                 "GET /derp HTTP/1.1\r\n\
                 Host: {host}\r\n\
                 Connection: Upgrade\r\n\
-                Upgrade: DERP\r\n\
+                Upgrade: WebSocket\r\n\
```

Comment:
Does derp servers already support basic websockets ?

Reply:
nat-lab tests are passing at least

Reply:
I've tested it with tcli and was able to establish derp connection and I think that during refinement @Jauler checked the derp server implementation also.